app/physicians/[slug]/page.tsx

```diff
@@ -1,155 +1,218 @@
-        ) => (
-          <ContentParagraph key={contentIndex} contentItem={contentItem} />
-        ),
-      )}
-    </div>
-  </div>
-);
-
 export default async function PhysicianBio({
   params,
 }: {
   params: { slug: string };
 }) {
   const docBio = await getPhysicianBioBySlug(params.slug);
+  const sectionTitles: string[] = [
+    "Medical School",
+    "Internship",
+    "Board Certification",
+    "Residency",
+    "State License",
+    "Fellowships",
+    "Affiliations",
+    "Awards and Recognitions",
+  ];
 
   let educationAndCertificatesHeaderRendered = false;
   let researchInsightsAndPublicationsHeaderRendered = false;
+
+  const isEducationAndCertificatesTitle = (title: string) =>
+    !educationAndCertificatesHeaderRendered && sectionTitles.includes(title);
+
+  const isResearchInsightsAndPublicationsTitle = (title: string) =>
+    !researchInsightsAndPublicationsHeaderRendered &&
+    (title === "Research Insights" || title === "Publications");
+
+  const physicianName = docBio.physicianName;
+  const physicianPortrait = docBio.physicianPortrait.fields.file.url;
+  const portraitWidth =
+    docBio.physicianPortrait.fields.file.details.image.width;
+  const portraitHeight =
+    docBio.physicianPortrait.fields.file.details.image.height;
+
+  const generateTargetID = (title: string): string => {
+    return `#${title.toLowerCase().split(" ").join("-")}`;
+  };
+
+  const options = [
+    { value: "overview", label: "Overview", targetID: "#overview" },
+    {
+      value: "educationAndCertifications",
+      label: "Education And Certifications",
+      targetID: generateTargetID("Education And Certificates"),
+    },
+    {
+      value: "researchInsightsAndPublications",
+      label: "Research Insights And Publications",
+      targetID: generateTargetID("Research Insights And Publications"),
+    },
+  ];
+
   return (
-    <main className="m-10">
-      <div>
-        <Image
-          src={docBio.physicianPortrait.fields.file.url}
-          alt=""
-          width={docBio.physicianPortrait.fields.file.details.image.width}
-          height={docBio.physicianPortrait.fields.file.details.image.height}
-        />
-        <h1 className="text-lg text-[#800080]">{docBio.physicianName}</h1>
-        <h3 className="text-base">Specializes in:</h3>
-        <div>
-          {renderRichTextToReactComponent(
-            docBio.specialties as unknown as Document,
-          )}
+    <main className="m-10 lg:max-w-screen-xl lg:mx-auto">
+      <div className="lg:grid lg:grid-cols-3 gap-2">
+        <div className="lg:h-96 h-72 mb-3">
+          <Image
+            src={physicianPortrait}
+            alt={physicianName}
+            width={portraitWidth}
+            height={portraitHeight}
+            style={{ objectPosition: "center 20%" }}
+            className="object-none h-full sm:w-auto "
+          />
+        </div>
+
+        <div className="NAME-AND-SPECIALTIES xl:-ml-24 lg:-ml-8 lg:mt-2">
```

Comment:
Since "NAME-AND-SPECIALTIES" is a comment, you can place it above the div and have it comment it out. Class name should just have the styles.